src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class ApplicationInsightsExtensions
     {
         public static IApplicationBuilder UseApplicationInsightsRequestTelemetry(this IApplicationBuilder app)
```

Comment:
It's certainly debatable. It's indeed a matter of how likely you think someone is to confuse this with something else or otherwise not understand what it is. The term "Middleware" in the method name is not required - it's implied.